spot_description/launch/description.launch.py

```diff
@@ -2,19 +2,45 @@
 
```

Comment:
I don't know if I'm setting something incorrectly, but I am getting this error when I try to run the Rviz with the URDF:
![image](https://github.com/bdaiinstitute/spot_ros2/assets/137220849/fe6c2a0b-a89e-4fe8-b200-418dcdaa47ba)

Reply:
I forgot to set a default for a parameter, thank you for catching this!